snyk/cli
# feat: remove support for paths outside the current working directory

#### What does this PR do?

This PR removes support for paths outside the current working directory for the `snyk iac test` command. This feature has been deprecated, the deprecation was announced, and the grace period for the removal terminated today.

#### How should this be manually tested?

Run the command with paths outside the current working directory, You will see a message similar to the following:

```
$ snyk iac test test/fixtures/iac/terraform ../foo ../bar

...

Test Failures

  Failed to parse Terraform file
  Path: test/fixtures/iac/terraform/sg_open_ssh_invalid_go_templates.tf
        test/fixtures/iac/terraform/sg_open_ssh_invalid_hcl2.tf

  Path is outside the current working directory
  Path: ../foo
        ../bar

...
```

#### What are the relevant tickets?

[CFG-1973](https://snyksec.atlassian.net/browse/CFG-1973)
